### DISCUSSION TITLE
Mappacks changing color on completion

### DISCUSSION THREAD

**Discussion by Chibu68** [ppy/osu#18000] [Ideas] [answered]
I don't know if it's been addressed already since the website is being redesigned, but I haven't seen anyone bring it up while doing a quick search on GitHub, so I thought I'd share it anyway. I think it would be cool if mappacks changed color upon completion, just like the clear circle. In my opinion, it would feel more rewarding to complete a mappack and it would become way easier to keep track of them. The images below are a suggestion as to what it could look like.
![810826ED-0EA3-4B6F-A205-E08F03285CAA](https://user-images.githubusercontent.com/62471871/165647329-d9d65e2d-51f0-4c5f-bfea-27b53ddc2bae.png)
![0C5D09C7-9D89-46DD-AA7E-614B68E2FA06](https://user-images.githubusercontent.com/62471871/165647335-bd511673-303c-4666-ae20-a012e6b4b38a.png)

**Comment by bdach** [comment, marked as answer]
Website issues/suggestions go on the website repository (https://github.com/ppy/osu-web).
